src/test/scala/com/cognite/spark/datasource/TimeSeriesRelationTest.scala

```diff
@@ -115,6 +116,7 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest {
          |select '$insertDescription' as description,
          |isString,
          |concat('TEST_', name) as name,
+         |concat('TEST_', name) as legacyName,
```

Comment:
Remove this from tests, we don't want to change the external schema and it doesn't look like we are. :+1: